step.yml

```diff
@@ -1,7 +1,34 @@
 title: Export iOS and tvOS Xcode archive
 summary: Export iOS and tvOS IPA from an existing Xcode archive
 description: |-
-  Exports an IPA from an existing iOS and tvOS `.xcarchive` file.
+  Manages your code signing assets and exports an IPA from an existing iOS and tvOS `.xcarchive` file. You can add multiple **Export iOS and tvOS Xcode archive** Steps to your Workflows to create multiple different signed .ipa files.
+
+  ### Configuring the Step
+  Before you start:
+  - Make sure you have connected your [Apple Service account to Bitrise](https://devcenter.bitrise.io/en/accounts/connecting-to-services/apple-services-connection.html).
+  Alternatively, you can upload certificates and profiles to Bitrise manually, then use the Certificate and Profile installer step before Xcode Archive
+  - Make sure certificates are uploaded to Bitrise's **Code Signing** tab. The right provisioning profiles are automatically downloaded from Apple as part of the automatic code signing process.
+
+  To configure the Step:
+  1. **Archive Path**: Specifies the archive that should be exported. The input value sets xcodebuild's `-archivePath` option.
+  2. **Select a product to distribute**: Describes which product to export.
```

Comment:
In case of the input, the value options are adding this context, but basically, here users need to `Decide if an App or an App Clip IPA should be exported`.